src/Components/ItemsList.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import UserInput from "./UserInput";
+import Items from "./Items";
+import { Button } from "react-bootstrap";
+
+export default class ItemsList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      itemsList: [],
+    };
+  }
+  //We need to be able to add items to the list
+  //Items are added in the UserInput component: when user inputs something into the input field and then clicks "add item" , the item should get added into the itemsArray
+  //Each item can be deleted
+  //Each item can be changed
+  //Each item can be edited
+  //Each item has checkbox, once the user got the item, they may tick the checkbox and the item will get crossed out
+
+  //1) adding items logic
+  //this function is passed as props to UserInput
```

Comment:
I can infer this from how the function is used in the code, so I do not need to document this in a comment